rali/rali/include/randombboxcrop_meta_data_reader.h

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright (c) 2019 - 2020 Advanced Micro Devices, Inc. All rights reserved.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+#pragma once
+
+#include <string>
+#include <memory>
+#include "randombboxcrop_meta_data.h"
+#include "reader.h"
+#include "meta_data_reader.h"
+#include "graph.h"
+#include "parameter_factory.h"
+
+enum class RandomBBoxCrop_MetaDataReaderType
+{
+    RandomBBoxCropReader = 0,
+};
+enum class RandomBBoxCrop_MetaDataType
+{
+    BoundingBox
+};
+
+struct RandomBBoxCrop_MetaDataConfig
+{
+private:
+    RandomBBoxCrop_MetaDataType _type;
+    RandomBBoxCrop_MetaDataReaderType _reader_type;
+    bool _all_boxes_overlap;
+    bool _no_crop;
+    FloatParam* _aspect_ratio;
+    bool _has_shape;
+    int _crop_width;
+    int _crop_height;
+    int _num_attempts;
+    FloatParam* _scaling;
+    int _total_num_attempts;
+public:
+    RandomBBoxCrop_MetaDataConfig(const RandomBBoxCrop_MetaDataType& type, const RandomBBoxCrop_MetaDataReaderType& reader_type, const bool& all_boxes_overlap,
+                        const bool& no_crop, FloatParam* aspect_ratio, const bool& has_shape, const int& crop_width, const int& crop_height, const int& num_attempts, FloatParam* scaling, const int& _total_num_attempts):  _type(type), _reader_type(reader_type),
```

Comment:
rename variable _total_num_attemps to total_num_attempts. The member variable has the same name as well.

Reply:
renamed _total_num_attemps to total_num_attemps